10.200.7.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.5.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.9.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.6.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.3.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.1.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.8.8 | SUCCESS => {
    "changed": true, 
    "dest": "C:\\soft\\PreventCopy\\", 
    "operation": "folder_copy", 
    "src": "/root/ansible_playbooks/preventcopy/sql_import_bat"
}
10.200.2.8 | UNREACHABLE! => {
    "changed": false, 
    "msg": "plaintext: HTTPConnectionPool(host='10.200.2.8', port=5985): Max retries exceeded with url: /wsman (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6f21824750>, 'Connection to 10.200.2.8 timed out. (connect timeout=30)'))", 
    "unreachable": true
}
10.200.4.8 | UNREACHABLE! => {
    "changed": false, 
    "msg": "plaintext: HTTPConnectionPool(host='10.200.4.8', port=5985): Max retries exceeded with url: /wsman (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6f21867950>, 'Connection to 10.200.4.8 timed out. (connect timeout=30)'))", 
    "unreachable": true
}
10.200.7.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.5.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.9.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.3.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.8.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.1.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.6.8 | SUCCESS | rc=0 >>
start running...
ok�?
�밴���������. . . 
mysql: [Warning] Using a password on the command line interface can be insecure.

10.200.4.8 | UNREACHABLE! => {
    "changed": false, 
    "msg": "plaintext: HTTPConnectionPool(host='10.200.4.8', port=5985): Max retries exceeded with url: /wsman (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f96392db0d0>, 'Connection to 10.200.4.8 timed out. (connect timeout=30)'))", 
    "unreachable": true
}
10.200.2.8 | UNREACHABLE! => {
    "changed": false, 
    "msg": "plaintext: HTTPConnectionPool(host='10.200.2.8', port=5985): Max retries exceeded with url: /wsman (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f96392bbe50>, 'Connection to 10.200.2.8 timed out. (connect timeout=30)'))", 
    "unreachable": true
}
